.ebextensions/00_ami.config

```diff
@@ -2,7 +2,7 @@ option_settings:
   aws:elasticbeanstalk:container:nodejs:
     NodeVersion: 6.9.1
     ProxyServer: apache
-    NodeCommand: 'node_modules/.bin/forever -a -m 10 start app.js'
+    NodeCommand: 'node_modules/.bin/forever -a -m 10 app.js'
```

Comment:
`forever start` daemonizes the process; but Elastic Beanstalk runs the Node command under upstart, which does its own version of daemonization and doesn't like PIDs switching on it. Removing the `start` tells forever to run the process in the foreground.